### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Apply the flexible alerts plugin
- Edit the architecture diagram to improve clarity
- Add `docs/README.md`
- Remove reference to an internal repository 
- Language editing

See [#18552](https://github.com/kyma-project/kyma/issues/18552)

### CODE REVIEW

File: docs/contributor/06-20-environments-cleanup-cronjob.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Environments Cleanup CronJob
 
->**CAUTION:** The CronJob must run only in the development environment. You must be connected to the development Kubernetes cluster before applying the CronJob.
+> [!WARNING] 
+> The CronJob must run only in the development environment. You must be connected to the development Kubernetes cluster before applying the CronJob.
 
->**NOTE:** Environments Cleanup CronJob is applied manually. There is no automatic release process for the job because of its destructiveness. To get more details on using the job, visit its [`README`](../../utils/kyma-environments-cleanup-job/README.md). 
+> [!NOTE] 
+> Environments Cleanup CronJob is applied manually. There is no automatic release process for the job because of its destructiveness. To get more details on using the job, visit its [`README`](../../utils/kyma-environments-cleanup-job/README.md).
```
Comment (nataliasitko) [inline docs/contributor/06-20-environments-cleanup-cronjob.md:7]:
```suggestion
> Environments Cleanup CronJob is applied manually. There is no automatic release process for the Job because of its destructiveness. To get more details on using the Job, visit its [`README`](../../utils/kyma-environments-cleanup-job/README.md). 
```

File: docs/user/03-20-runtime-operations.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ Kyma Environment Broker (KEB) allows you to configure operations you can run on
 - Return an error, which interrupts the entire process, or skip step execution.
 - Repeat the entire operation after a specified period.
 
-> **NOTE:** It's important to set lower timeouts for the Kyma installation in the Runtime Provisioner.
+> [!NOTE] 
+> It's important to set lower timeouts for the Kyma installation in the Runtime Provisioner.
```
Comment (nataliasitko) [inline docs/user/03-20-runtime-operations.md:9]:
```suggestion
> It's important to set lower timeouts for the Kyma installation in Runtime Provisioner.
```

File: docs/contributor/02-60-orchestrate-kyma-upgrade.md [hunk 3]
```diff
@@ -36,52 +58,30 @@ This tutorial shows how to upgrade SAP BTP, Kyma runtime using Kyma Environment
...
+      ```json
+      {
+          "orchestrationID":"8a7bfd9b-f2f5-43d1-bb67-177d2434053c"
+      }
+      ```
 
 4. [Check the orchestration status](02-70-orchestration-status.md).
 
->**NOTE:** Only one orchestration request can be processed at the same time. If KEB is already processing an orchestration, the newly created request waits for processing with the `PENDING` state.
+> **NOTE:** Only one orchestration request can be processed at the same time. If KEB is already processing an orchestration, the newly created request waits for processing with the `PENDING` state.
```
Comment (nataliasitko) [inline docs/contributor/02-60-orchestrate-kyma-upgrade.md:87]:
```suggestion
   >**NOTE:** Only one orchestration request can be processed at the same time. If KEB is already processing an orchestration, the newly created request waits for processing with the `PENDING` state.
```

File: docs/contributor/02-10-runtime-components.md [hunk 4]
```diff
@@ -47,7 +48,7 @@ type OptionalComponentDisabler interface {
 	Disable(components internal.ComponentConfigurationInputList) internal.ComponentConfigurationInputList
 ```
 
->**TIP**: Check the [CustomDisablerExample](../../internal/runtime/custom_disabler_example.go) as an example of custom service for disabling components.
+> **TIP:** Check the [CustomDisablerExample](../../internal/runtime/custom_disabler_example.go) as an example of custom service for disabling components.
```
Comment (nataliasitko) [inline docs/contributor/02-10-runtime-components.md:51]: This tip and previous notes are not indented correctly, I think

**Review by nataliasitko — APPROVED** [review]

File: docs/contributor/02-10-runtime-components.md [hunk 5]
```diff
@@ -1,12 +1,13 @@
...
-> **NOTE:** Once all Kyma components become independent modules, Kyma Environment Broker will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
+> [!NOTE] 
+> Once all Kyma components become independent modules, Kyma Environment Broker will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
 
 Kyma Environment Broker (KEB) serves the functionality of composing the list of components that are installed in SAP BTP, Kyma runtime. The diagram and steps describe the KEB workflow in terms of calculating and processing Kyma runtime components:
 
 ![runtime-components-architecture](../assets/runtime-components.svg)
 
-1. The user provisions SAP BTP, Kyma runtime and selects optional components that they want to install.
+1. The user provisions Kyma runtime and selects optional components that they want to install.
```
Comment (MarekMichali) [inline docs/contributor/02-10-runtime-components.md:10]: Was this change intentional?

**Review by MarekMichali — APPROVED** [review]
